components/layout/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import Header from '../header'
+import Head from '../head'
+import './index.css'
+import Sidebar from '../sidebar'
+
+const Layout = ({ children }) => {
+  return (
+    <>
+      <Head />
+      <div className="layout">
+        <Header className="layout-bar fixed" />
```

Comment:
You removed mobile behaviour at all, right?

Reply:
yes will add it in next PR